pkg/filesystem/fs/fs.go

```diff
@@ -62,9 +62,15 @@ var nativeEndian binary.ByteOrder
 type Mode int
 
 const (
+	// A single nydusd serves all container images
 	SharedInstance Mode = iota
+	// One container image one nydusd topology
 	MultiInstance
+	// Rafs is integrated with other custom components.
+	// Nydusd is not needed to work as FUSE server or other backend service.
```

Comment:
Nydusd is not needed to work as FUSE server (fusedev) or other fs drivers, just provide the mount slices to containerd, and do real mount by other components.

Reply:
fixed